pom.xml

```diff
@@ -56,7 +56,9 @@
       <module>rhq-plugin</module>
       <module>osgi</module>
       <module>spring/spring</module>
-      <module>spring/spring4</module>
```

Comment:
Yes, that's intentional. `spring/spring4/pom.xml` is a parent pom for Spring 4 modules which doesn't need to be built at all. It just makes things a little bit easier when importing the project into IDE.